.github/workflows/build.yml

```diff
@@ -193,14 +199,14 @@ jobs:
         env:
           PR: ${{ github.event.pull_request.number }}
       - name: Build
-        run: yarn ember build --environment production
+        run: pnpm ember build --environment production
       - name: Pack
         run: |
-          VERSION="$(jq -r '.version | split("+") | .[0]' package.json)"
```

Comment:
Why was this changed?

Reply:
So the new one will work with all possible version types? Not sure if this is also needed for nightly versions etc

Reply:
Well, the version string during the t